File: src/1051.height-checker.1137/solution.ts
function heightChecker (heights: number[]): number {
    const expected = [...heights, ].sort((a, b) => a - b);
    let result = 0;
    for (let i = 0; i < heights.length; i++) {
        if (heights[i] !== expected[i]) {
            result++;
        }
    }
    return result;
}

File: src/1051.height-checker.1137/solution.java
class Solution {
    public int heightChecker(int[] heights) {
        int[] copy = Arrays.copyOf(heights, heights.length);
        Arrays.sort(heights);
        int count = 0;
        for (int i = 0; i < heights.length; i++) {
            if (heights[i] != copy[i]) {
                count++;
            }
        }
        return count;
    }
}